src/wp-includes/js/customize-preview-widgets.js

```diff
@@ -220,12 +242,15 @@ wp.customize.widgetsPreview = wp.customize.WidgetCustomizerPreview = (function(
 		},
 
 		/**
-		 * Get the before/after boundary nodes for all instances of this sidebar (usually one).
+		 * Gets the before/after boundary nodes for all instances of this sidebar (usually one).
 		 *
 		 * Note that TreeWalker is not implemented in IE8.
 		 *
 		 * @since 4.5.0
-		 * @returns {Array.<{before: Comment, after: Comment, instanceNumber: number}>}
+		 *
+		 * @returns {Array.<{before: Comment, after: Comment, instanceNumber: number}>|void} Either an array with an object
```

Comment:
I think `void` is wrong, because the only return is used in a callback in `__.each` and that is in a closed scope. So: it only returns an array.